.github/workflows/maven.yml

```diff
@@ -0,0 +1,24 @@
+# This workflow will build a Java project with Maven
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: Java CI with Maven
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
could you please setup this for opened and synchronized pull request as well?

Reply:
probably like adding 
```
  pull_request:
    types: [opened, synchronize]
```

Reply:
Yes. Do You need any other type? 
https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#pull_request

Reply:
for now it's good. In next iteration maybe I'll work upon it.
Thanks for the PR :+1: